packages/stockflux-components/src/components/icons/minimize.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="12px" height="2px" viewBox="0 0 12 2" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 59 (86127) - https://sketch.com -->
```

Comment:
Nitpick; these comments could be removed from `minimize`, `link`, `close` and `watchlist` to keep them consistent with the other cut down files?

Reply:
Removed comments and bloat from all svgs so far.